mytoyota/models/endpoints/trips.py

```diff
@@ -66,6 +105,27 @@ class _HDCModel(BaseModel):
     power_time: Optional[int] = Field(alias="powerTime", default=None)
     power_dist: Optional[int] = Field(alias="powerDist", default=None)
 
+    def __add__(self, other: _HDCModel):
```

Comment:
Could you please explain what exactly this is needed for.
Please also add a few unit tests to maintain the coverage. I think these could also contribute to the explanation.

Reply:
These are required as the only way to get the Monthly and Yearly summaries is too add together either the Daily summaries or the Monthly Summaries, seemed the best place to add them.